Tests/SentryTests/Integrations/Feedback/SentryFeedbackTests.swift

```diff
@@ -30,4 +55,107 @@ class SentryFeedbackTests: XCTestCase {
         let attachments = sut.attachments()
         XCTAssertEqual(attachments.count, 0)
     }
+        
```

Comment:
`l`: instead of creating a single test which loops over all input combinations, you could create individual tests for each input combination, but each one calls a helper function. The helper function would then pass `...file: StaticString = #file, line: UInt = #line)` as parameters down to the `XCTAssertEqual(..., file: file, line: line)`, therefore propagating the error up to the calling function.

That would then result in individual tests failing if the input combination is not working, rather than failing the whole

Reply:
i did consider dynamically generating a test function per case, if this gets painful enough i will spend more time on it later